protoc-gen-twirp/generator.go

```diff
@@ -1078,7 +1097,7 @@ func (t *twirp) generateServerRouting(servStruct string, file *descriptor.FileDe
 	t.P(`// `, pathPrefixConst, ` is used for all URL paths on a twirp `, servName, ` server.`)
 	t.P(`// Requests are always: POST `, pathPrefixConst, `/method`)
 	t.P(`// It can be used in an HTTP mux to route twirp requests along with non-twirp requests on other routes.`)
-	t.P(`const `, pathPrefixConst, ` = `, strconv.Quote(pathPrefix(file, service)))
+	t.P(`const `, pathPrefixConst, ` = `, strconv.Quote(pathPrefix(file, stringutils.CamelCase(service.GetName()))))
```

Comment:
This is the same as using `servName`, but it's fine, being explicit may be more readable.

Reply:
Yes I've been trying to keep strings and go variables separate so that later one can change these easier if necessary.